README.md

```diff
@@ -400,6 +422,14 @@ You can specify a fixed, valid value to replace a target FHIR field. For example
 For complex data types, you can provide a fixed json fragment following the [sample rules](#Sample-rules-using-FHIRPath).
 You should provide valid value for the target data type to avoid unexpected errors.
 
+## Perturb method
+With perturbation rule, you can replace specific values with equally specific, but different values. You can choose to sample noise from a fixed range or a proportional range. There are a few parameters that can help you customize the noise amount for different FHIR types.

```

Comment:
Explain with examples what is fixed range and what is a proportional range

Reply:
Thanks, updated